<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LinesPreferencesWidget</class>
 <widget class="QWidget" name="LinesPreferencesWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>265</width>
    <height>166</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Lines Preferences</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="sideToolbarGroupBox">
     <property name="title">
      <string>Side Toolbar position</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QRadioButton" name="sideToolbarLeftRadioButton">
        <property name="text">
         <string>Left</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">sideToolbarPositionbuttonGroup</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="sideToolbarRightRadioButton">
        <property name="text">
         <string>Right</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">sideToolbarPositionbuttonGroup</string>
        </attribute>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Side toolbar width</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="sideToolbarWidthSpinBox">
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Main toolbar Height</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="mainToolbarHeightSpinBox">
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>2</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="sideToolbarPositionbuttonGroup"/>
 </buttongroups>
</ui>
